common/devdoc/interlocked_hl_requirements.md

```diff
@@ -74,6 +80,26 @@ MOCKABLE_FUNCTION_WITH_RETURNS(, INTERLOCKED_HL_RESULT, InterlockedHL_WaitForVal
 
 **SRS_INTERLOCKED_HL_01_006: [** If `wait_on_address` fails, `InterlockedHL_WaitForValue` shall fail and return `INTERLOCKED_HL_ERROR`. **]**
 
+###  InterlockedHL_WaitForValue64
+
+```c
+MOCKABLE_FUNCTION_WITH_RETURNS(, INTERLOCKED_HL_RESULT, InterlockedHL_WaitForValue64, int64_t volatile_atomic*, address_to_check, int64_t, value_to_wait, uint32_t, timeout_ms)(INTERLOCKED_HL_OK, INTERLOCKED_HL_ERROR);
```

Comment:
Arguments have different names here vs the 32-bit API. 

I think its better if we stick to the same names so that if we want to refactor later, it is easier.

Reply:
Comment applies to all 64 bit APIs

Reply:
I am waiting on @dcristoloveanu for this since when I used this function for the first time, it was a bit difficult to understand just looking at the signatures. I am more inclined towards updating the 32 bit version with similar notations so that the function calls are self explanatory.

Reply:
For example the following is not clear in the code or here
If the value at `address` is equal to `value`. 
Some function parameter were named `milliseconds`.

Reply:
I do not see any real issue with the name being `value` or `value_to_wait`. It's a bit of splitting hairs over it and I don't think it matter that much in this case. Just pick a choice and go with it for both APIs, makins dure it is consistent. I think that's what Parth was also flagging - that they should be consistent.

Reply:
This is corrected